tests/app/test_config.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from flask import Flask
+
+from app.config import Config
+
+
+@pytest.mark.parametrize(
+    ["api_host_name", "frontend_host_name", "expected_cookie_domain"],
+    [
+        # prod
+        ("https://documents.download.service.gov.uk", "https://download.service.gov.uk", "download.service.gov.uk"),
+        # notifications-local docker-compose
+        (
+            "http://api.document-download.localhost",
+            "http://frontend.document-download.localhost",
+            "document-download.localhost",
+        ),
+        # running locally outside of docker is no longer supported? :thinking_face:
+        pytest.param(
+            "http://localhost:7001", "http://localhost:7002", None, marks=pytest.mark.xfail(raises=ValueError)
```

Comment:
i suspect that the cookies never worked previously, but it only happened at cookie-setting-time. (which is only if you have confirm emails set to true)

now your app config won't be able to load which is not great. i might have a bit of a think